<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Jayaprakash Jayashankar">

<title>Tutorial on MS-Dial 5 – MS-DIAL 5 tutorial</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">MS-DIAL 5 tutorial</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./lipidomics.html"> 
<span class="menu-text">Lipidomics</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./metabolomics.html"> 
<span class="menu-text">Metabolomics</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./proteomics.html"> 
<span class="menu-text">Proteomics</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./imaging.html"> 
<span class="menu-text">Mass spectrometry imaging</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-utilities" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Utilities</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-utilities">    
        <li>
    <a class="dropdown-item" href="./utils.html">
 <span class="dropdown-text">Other utilities</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./duplicate_alignment_spotfeature.html">
 <span class="dropdown-text">Duplicate alignment spot feature to separate isomers</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./Determining MS data types.html">
 <span class="dropdown-text">Determining Mass Spectrometry (MS) Data Types in MS-Dial: Centroid or Profile</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./Determine the optimal minimum peak height.html">
 <span class="dropdown-text">Determining optimal minimum peak height</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./Explanation on each parameter of MS-Dial 5.html">
 <span class="dropdown-text">Tutorial on MS-Dial 5</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#introduction-to-ms-dial" id="toc-introduction-to-ms-dial" class="nav-link active" data-scroll-target="#introduction-to-ms-dial">1.0 Introduction to MS-Dial</a></li>
  <li><a href="#importance-of-ms-dial-5" id="toc-importance-of-ms-dial-5" class="nav-link" data-scroll-target="#importance-of-ms-dial-5">2.0 Importance of MS-Dial 5</a></li>
  <li><a href="#the-differences-between-ms-dial-4-and-ms-dial-5" id="toc-the-differences-between-ms-dial-4-and-ms-dial-5" class="nav-link" data-scroll-target="#the-differences-between-ms-dial-4-and-ms-dial-5">3.0 The differences between MS-DIAL 4 and MS-DIAL 5:</a></li>
  <li><a href="#step-by-step-guide-on-how-to-set-up-a-project-in-ms-dial-5" id="toc-step-by-step-guide-on-how-to-set-up-a-project-in-ms-dial-5" class="nav-link" data-scroll-target="#step-by-step-guide-on-how-to-set-up-a-project-in-ms-dial-5">4.0 Step-by-step guide on how to set up a project in MS-DIAL 5</a></li>
  <li><a href="#how-to-determine-profile-data" id="toc-how-to-determine-profile-data" class="nav-link" data-scroll-target="#how-to-determine-profile-data">How to determine profile data?</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Tutorial on MS-Dial 5</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Jayaprakash Jayashankar </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<section id="introduction-to-ms-dial" class="level2">
<h2 class="anchored" data-anchor-id="introduction-to-ms-dial">1.0 Introduction to MS-Dial</h2>
<p>MS-DIAL is a powerful, open-source software tool designed for the comprehensive analysis of mass spectrometry data, whether for untargeted metabolomics or lipidomics. One of its standout features is its compatibility with a wide range of instruments, including GC/MS, GC/MS/MS, LC/MS, and LC/MS/MS systems, and support for major vendors such as Agilent, Bruker, LECO, Sciex, Shimadzu, Thermo, and Waters. In this latest version, MS-DIAL enhances the user experience by eliminating the need for data format conversion into the ABF format—although it is still supported. Users can now directly import raw MS data. Still, the data can now be imported directly to MS-Dial. MS-DIAL provides a comprehensive ‘start-up kit’ that includes several MSP files with both EI- and MS/MS spectra to facilitate ease of use. MS-DIAL includes an internal version of the Fiehn lab’s GC/MS database, oriented by the FAME RI index, as well as an in-silico retention time- and MS/MS database for LC/MS/MS-based lipidomics. Additionally, MS-DIAL facilitates isotope-labeled tracking in LC/MS projects, making it a powerful tool for detailed metabolic studies and precise quantification. Its user-friendly graphical interface combined with broad functionality makes it a key resource for researchers conducting multi-omics studies (Figure 1).</p>
<div id="fig-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Determine MS profile or centroid data/Figure 1_Website.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: MS-Dial screenshot
</figcaption>
</figure>
</div>
</section>
<section id="importance-of-ms-dial-5" class="level2">
<h2 class="anchored" data-anchor-id="importance-of-ms-dial-5">2.0 Importance of MS-Dial 5</h2>
<p>The latest version of MS-DIAL 5 aims to be the most versatile tool for multi-omics data analysis and therefore supports direct injection, many separation (GC, LC, CE, SFC), ionization (EI, ESI, MALDI), fragmentation (CID, HCD, ETD, ECD, EIEIO, EID, OAD) and MS/MS (DDA, SWATH, AIF) datasets. In addition, technological advances in fragmentation mechanisms such as electron-impact excitation of ions from organics (EIEIO) have also been taken into account, making it possible to work with these types of data to offer comprehensive structural identification of metabolites, lipids, and proteins. These resources enhance the identification and quantification of metabolites and lipids. After data processing which includes peak picking, deconvolution, compound identification, and peak alignment, MS-DIAL provides several normalization methods (including LOWESS) and a multivariate analysis by principal component analysis (PCA). Finally, for further analysis by other programs, this program can export your result as table format (for SIMCA-P, MetaboAnalyst, and MetFamily etc.), and as several spectral formats including NIST, MassBank, and Mascot formats for compound identifications by MS-FINDER, CSI:FingerID, CFM-ID, MetFrag, and MetFamily etc.</p>
</section>
<section id="the-differences-between-ms-dial-4-and-ms-dial-5" class="level2">
<h2 class="anchored" data-anchor-id="the-differences-between-ms-dial-4-and-ms-dial-5">3.0 The differences between MS-DIAL 4 and MS-DIAL 5:</h2>
<p><strong>Graphical User Interface (GUI):</strong> MS-Dial 4: Functional but slightly less refined. MS-Dial 5: Improved GUI with more intuitive workflows and better user experience.</p>
<p><strong>Speed and Performance:</strong> MS-Dial 4: Effective but slower on larger datasets. MS-Dial 5: Optimized for faster data processing, especially for large, multi-omics datasets.</p>
<p><strong>Annotation Capabilities:</strong> MS-DIAL 4: Limited to using one database and annotation level in a single analysis. Provides comprehensive annotation of metabolites and lipids through spectral matching using established databases. MS-DIAL 5: Enables the use of multiple databases and annotation levels within the same analysis. Significantly improves annotation accuracy by incorporating ion mobility spectrometry (IMS) data and enhanced spectral deconvolution. MS-DIAL 5 also has better integration with larger and more updated databases, improving the annotation of complex molecules like lipids, metabolites, and isomers.</p>
<p><strong>Annotation Capabilities:</strong> MS-DIAL 4: Offers only one possible annotation candidate per analysis. MS-DIAL 5: Provides multiple annotation candidates from different databases, improving identification accuracy. Adds support for ion mobility spectrometry (IMS), enabling better separation of ions that share the same mass-to-charge ratio (m/z) but have different drift times. This improves separation in complex biological mixtures.</p>
<p><strong>Isomer Separation:</strong> MS-Dial 4: It will support basic functionality. MS-Dial 5: Enhanced separation of isomers and similar molecular species.</p>
<p><strong>Peak Picking and Alignment:</strong> MS-Dial 4: Accurate, but sometimes slower or less accurate with noisy data. MS-Dial 5: Refined peak picking and alignment algorithms for better performance, especially with complex datasets.</p>
<p><strong>Data Formats:</strong> MS-Dial 4: Supports only (.ABF) format. MS-Dial 5: Further extended support for newer formats, including advanced mass spectrometry data.</p>
<p><strong>Separation Techniques:</strong> MS-DIAL 4: Supports separation methods like LCMS, LCIMMS, and GCMS. MS-DIAL 5: Adds support for direct infusion MS (DIMS) and ion mobility MS (IMMS), broadening the range of separation techniques.</p>
<p><strong>Collision Types:</strong> MS-DIAL 4: Works with CID and HCD collision types. MS-DIAL 5: Extends support to additional collision types, including ECD, HotECD, EIEIO, EID, and OAD, offering more advanced fragmentation methods.</p>
<p><strong>Targeted Omics:</strong> MS-DIAL 4: Focuses on metabolomics and lipidomics analysis. MS-DIAL 5: Expands its capabilities to include proteomics, in addition to metabolomics and lipidomics.</p>
<p><strong>Flexibility in MS Methods:</strong> MS-DIAL 4: Only one MS method (DDA, SWATH, AIF) can be applied in each analysis. MS-DIAL 5: Allows multiple MS methods to be applied to a single analysis, enabling more comprehensive data analysis.</p>
<p><strong>Undo/Redo Functionality:</strong> MS-DIAL 4: Does not offer an undo/redo feature when changing an annotation to unknown. MS-DIAL 5: Introduces this capability, making it easier to revert or correct annotation changes during data processing.</p>
</section>
<section id="step-by-step-guide-on-how-to-set-up-a-project-in-ms-dial-5" class="level2">
<h2 class="anchored" data-anchor-id="step-by-step-guide-on-how-to-set-up-a-project-in-ms-dial-5">4.0 Step-by-step guide on how to set up a project in MS-DIAL 5</h2>
<p>This tutorial demonstrates how to start a new project in MS-DIAL 5, covering the steps to import raw data and set the MS method type. To begin, launch MS-DIAL 5 from your desktop or application menu, which will open the main user interface with various options and toolbars. The first step is to create a new project by setting up a dedicated folder where all project files, including raw data and results, will be stored. After selecting New Project from the “File” menu, choose the designated folder to ensure data is organized. Next, you can import raw data directly into MS-DIAL 5 without the need for conversion to ABF format. After selecting your raw data files, proceed to configure the MS method type according to your experiment, ensuring the appropriate settings are in place for accurate data analysis. This streamlined process sets the foundation for efficient data processing in MS-DIAL 5 (Figure 2).</p>
<div id="fig-2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Determine MS profile or centroid data/Figure 1_Website.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Start up a project in MS-DIAL5. First, select a new project to be created (A). Next, click browse to select a location where your project files will be saved (B). Select the folder where you want to save the MS-Dial project (C). Click run to go to the next parameter setting (D)
</figcaption>
</figure>
</div>
<p>After selecting the project folder, click on Browse in the raw measurement file section, and navigate to the folder where your raw data is stored. Select the raw files you wish to analyze, or simply drag and drop your files (including .d format files) directly into MS-DIAL 5. Once you’ve selected the appropriate files, click Open to import them. After importing your raw data, you can further assign identifiers to your measurements based on sample type (e.g., Sample, Standard, QC, Blank). You can also categorize them by class ID, according to your experimental design. At this point, you need to set the acquisition type for each raw data file.</p>
<p><strong>Here’s how to set the MS method type:</strong></p>
<p><strong>DDA (Data-Dependent Acquisition):</strong> Select this if you are analyzing single MS or data-dependent MS/MS data.</p>
<p><strong>DIA (Data-Independent Acquisition):</strong> Choose this option when analyzing SWATH or AIF data, where MS/MS data is acquired independently of precursor ions.</p>
<p>For a detailed walkthrough on selecting the acquisition type, refer to the video linked below, which demonstrates the process visually.</p>
<div class="quarto-video ratio ratio-16x9"><iframe data-external="1" src="https://www.youtube.com/embed/2iy-fsbusYI?si=49SP7rO1YIfv1BUO" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
<p>After selecting the acquisition type, you can further customize your project by setting additional parameters. If your analysis involves multiple batches, you can assign batch numbers to differentiate between them and account for batch effects during data processing. You can also specify the analytical order of your samples, which is useful for tracking trends such as signal drift or other variations that may have occurred throughout the run. Additionally, you can set a dilution factor for each sample, allowing MS-DIAL to adjust the quantification accordingly. If certain samples are not needed for further analysis, you can choose to exclude them from the data processing (figure 3).</p>
<div id="fig-3" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Determine MS profile or centroid data/Figure 1_Website.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: Click on the browse icon and point MS-DIAL5 to the folder where your raw data is stored (A). Next, select the raw file format as per your analysis (B). Select the raw files for analysis (C). Further, assign identifiers to your measurements and acquisition type (D)
</figcaption>
</figure>
</div>
<p>After assigning identifiers to the raw measurement files, click Next to proceed to the Measurement Parameters tab. In this section, you will input the analytical and instrumental parameters that were used during data acquisition. These settings are crucial for ensuring accurate data processing and should align exactly with the experimental conditions under which the measurements were taken. The following sections will offer detailed instructions on how to select and configure each parameter within the Measurement Parameters section</p>
<p><strong>4.1 Measurement Parameters Section:</strong></p>
<p><strong>1. Ionization type:</strong> Select the soft ionization type, when you are analyzing the ESI data or select hard ionization type when you are analyzing the EI data.</p>
<p><strong>2. Separation type:</strong> Select chromatography (for LC, GC, CE, or SFC) or direct infusion method as per your analysis. Additionally, you can check ion mobility box coupled with one of the previous options.</p>
<p><strong>3. Collision type:</strong> Select the corresponding collision type according to your analysis.</p>
<p><strong>4. Data type (MS1):</strong> Select the profile or centroid data as per your analysis in MS1.</p>
<p><strong>5. Data type (MS/MS):</strong> Select the profile or centroid data as per your analysis in MS/MS.</p>
<p>*Refer to the tutorial on MS-Dial on how to determine profile or centroid data type for your analysis using the Rawdataviewer tool in the below link.</p>
{{&lt; https://systemsomicslab.github.io/msdial5tutorial/Determining%20MS%20data%20types.html  &gt;}}
<p><strong>6. Ion mode:</strong> Select the corresponding ion mode according to your analytical setup.</p>
<p><strong>7. Target Omics:</strong> Select what kind of omics data you want to perform. If you select the ‘lipidomics’ project, you do not have to prepare NIST MSP format library since MS-DIAL internally contains the theoretical MS/MS spectra of lipids (and actually, the fragment ions are future evaluated by the decision tree algorithm to provide the proper lipid structure representation). On the other hand, when you select the ‘metabolomics’ project, your own MSP file will be required for compound identification.</p>
<p><strong>8. Advanced option:</strong> You can provide additional information regarding your experiment. This metadata will be included in the MassBank, MoNA and NIST (MSP) format export.</p>
<div id="fig-4" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Determine MS profile or centroid data/Figure 1_Website.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: Overview of the Measurement parameter settings
</figcaption>
</figure>
</div>
<p>After configuring each parameter in the Measurement Parameters section, click “Next” to proceed to the Data Collection tab. In this section, you will need to enter the data collection parameters that were used during the acquisition of your samples. These parameters are essential for accurately processing and analyzing your data according to the conditions of your experiment. The upcoming sections will provide detailed instructions on how to set each parameter in the Data Collection section.</p>
<p><strong>4.2 Data collection parameters:</strong></p>
<p><strong>Mass accuracy (Centroid parameter):</strong> After the initial peak detection in MS-DIAL, mass tolerance defines the acceptable range of mass deviation for both MS1 and MS/MS data. By default, mass tolerance values are set to ±0.01 Da for MS1 and ±0.025 Da for MS/MS. This means that peaks within these ranges around the detected mass are integrated and considered for further analysis. Importantly, the MS/MS tolerance also influences how MS/MS chromatograms are constructed, affecting how closely the detected fragment ions need to match the theoretical masses to be included in the MS2Dec deconvolution process.</p>
<p><strong>Data collection parameters:</strong> In MS-DIAL, you can define the analysis ranges as per your analysis for retention time (RT), MS1, and MS/MS axis to focus on specific parts of your data. For this demonstration, we have chosen the expected data range as follows: In this demo data RT range is 0 - 100 min, and the MS and MS/MS range is 0 – 2000 Da.</p>
<p><strong>Execute Retention Time Correction:</strong> This setting adjusts for any shifts or drifts in retention times that may occur during the analysis, ensuring that peaks are aligned accurately across different samples or runs. Use this function when you analyze your data on different elution conditions (eg., different columns).</p>
<p><strong>Isotope Recognition:</strong> Isotope recognition is used to identify and handle isotopic peaks that correspond to the same molecular ion but with different masses due to natural isotopic variations (e.g., [M+1] or [M+2]). This process helps in accurately quantifying the main molecular ion by distinguishing it from its isotopic counterparts. As long as you focus on small molecule research (less than 2000 Da), the maximum charged number can be set to 2. On the other hand, the parameter can be changed to 8 or more to process proteome or snRNA research data.</p>
<p><strong>Consider Cl and Br Elements:</strong> If you assume your samples might contain chlorine (Cl) or bromine (Br), enable this option as these elements have unique patterns in their isotopic distribution. This setting helps to accurately identify and quantify compounds by recognizing the specific isotopic peaks that Cl and Br create.</p>
<p><strong>Multithreading:</strong> Please set the count of threads that you want to use. You can check the maximum thread counts in resource monitor. (open task manager-&gt;open resource monitor).</p>
<div id="fig-5" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Determine MS profile or centroid data/Figure 1_Website.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5: Overview of the Data collection parameter setting.
</figcaption>
</figure>
</div>
<p>Click Next to proceed to the Peak Detection Parameter Settings. In this section, you’ll need to enter the parameters used for peak detection in your mass spectrometry data. This includes specifying values such as the minimum peak height, mass peak width, and exclusion mass list. The following sections will offer detailed guidance on how to configure each parameter within the Peak Detection section</p>
<p><strong>4.3 Peak detection parameter:</strong></p>
<p>MS-DIAL provides two simple thresholds; minimum values for peak width and height. Peaks below these thresholds are ignored.</p>
<p><strong>Minimum Peak Height:</strong> Set the minimum intensity level required for a peak to be detected in your data. Peaks with heights below this threshold will be ignored, which helps in filtering out noise and focusing only on significant signals for accurate analysis. Ideally, users put values here based on their own experience that you are looking at the trend of your data. However, based on our experience, the minimum peak height may be set to 1000 as a default value for this demo data. Besides, for FT-MS or Orbitrap data, the minimum peak height maybe 10,000 or more.</p>
<p>Refer to the tutorial on how to determine the minimum peak height for your analysis using the Rawdataviewer tool in the link below.</p>
{{&lt; https://systemsomicslab.github.io/msdial5tutorial/Determine%20the%20optimal%20minimum%20peak%20height.html  &gt;}}
<p><strong>Mass Slice Width:</strong> Define the range of mass-to-charge ratios (m/z) around each detected peak that will be considered during data analysis. This width helps in grouping ions within a specified m/z range to improve peak detection and quantification by accounting for slight variations in mass measurements. In this case, we are using 0.1 Da. From our experience, 0.1 and 0.05 are suitable for Q-TOF and Orbitrap, respectively.</p>
<p><strong>Smoothing method:</strong>Linear-weighted moving average is used for the peak detection as default to accurately determine the peak left- and right edges.</p>
<p><strong>Smoothing level:</strong> The recommended smoothing level is 1-3. It is the number of points on each side of the peaks.</p>
<p><strong>Minimum peak width:</strong> Minimum peak width indicates a threshold of peak width for filtering. For example., we choose a minimum peak width set to 5 scans, peaks that span less than 5 scans will be excluded from the results. Only peaks with a width of 5 scans or greater will be included in the analysis.</p>
<p><strong>Exclusion mass list:</strong> If you already know unwanted m/z peaks because of columns or solvent contaminants, you can specify them to exclude those peaks.</p>
<div id="fig-6" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Determine MS profile or centroid data/Figure 1_Website.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6: Overview of the Peak detection parameter settings.
</figcaption>
</figure>
</div>
<p>In the RawDataViewer tool, click on browse and select any raw file to check whether the MS data is centroid or profile and click open. Once the file is loaded into the software. In the Method Type section, choose the appropriate method based on your analysis, and select the Ion Mode (positive or negative) to match your sample. After configuring these settings, click Load to upload the file. Once the file is loaded, select the file and click the show button to display the data. Once you click the show button, a new window will appear displaying a histogram of peak height, MS, and MS/MS peak intensity information. This window also provides a table with details such as scan start time, polarity, MS levels, base peak m/z, and base peak intensity for each metabolite in your sample (Figure 1).</p>
<div id="fig-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Determine MS profile or centroid data/Figure 1_Website.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;7: Open the RawDataViewer application (A). Browse and select the raw MS file to determine whether the data is centroid or profile (B). Click the load button to load the raw file into the tool (C). Click the show button to display the histograms of peak information
</figcaption>
</figure>
</div>
<p>To organize the data, double-click on the base peak intensity column to sort the lipid list in ascending order based on MS levels and peak intensity. To determine whether the MS1 data is centroid or profile, click on the most intense MS1 peak (typically the first peak in the table). Right-click on the peak and zoom to examine the base peak of the selected peak. If the peak appears as a single sharp peak with its m/z value centered at the peak’s highest point, it indicates that the data is in centroid mode. Next, scroll through the table to examine the MS/MS (MS2) data. Select the MS2 peak in the table, right-click, and zoom in on the base peak of the selected peak. The MS2 peak also shows a single, high-intensity peak centered at its m/z value, this confirms that both the MS1 and MS/MS data are in centroid mode (Figure 2).</p>
<div id="fig-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Determine MS profile or centroid data/Figure 2_Website.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;8: Click on the base peak intensity column to sort the peak list in ascending order based on MS levels and peak intensity (A). Select the MS1 peak (usually the first peak in the table) (B). Zoom the MS1 base peak (C). Select and zoom in on the MS2 base peak (D)
</figcaption>
</figure>
</div>
</section>
<section id="how-to-determine-profile-data" class="level2">
<h2 class="anchored" data-anchor-id="how-to-determine-profile-data">How to determine profile data?</h2>
<p>Next, open the raw file and follow the same steps until a new window of histograms of peak height, MS, and MS/MS peak intensity of the selected raw file appears. Double-click on the base peak intensity column to sort the lipid list in ascending order based on MS levels and peak intensity. Select the first peak (MS1) in the table, then right-click, and zoom in on the base peak of the selected peak. The peak appears like a detailed view of the entire peak shape by showing the continuous intensity distribution of the peak across a range of m/z values. it indicates that the data is in profile mode. To check the MS/MS data, scroll down the table. Select the MS2 level peak, then right-click, and zoom in on the base peak of the selected peak. The MS2 peak also shows a continuous distribution of peak intensities, this confirms that both the MS1 and MS/MS data are in profile mode (Figure 3).</p>
<div id="fig-1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/Determine MS profile or centroid data/Figure 3_Website.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;9: Click on the base peak intensity column to sort the peak list in ascending order based on MS levels and peak intensity (A). Select the MS1 peak (usually the first peak in the table) (B). Zoom the MS1 base peak (C). Select and zoom in on the MS2 base peak (D)
</figcaption>
</figure>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/systemsomicslab\.github\.io\/msdial5tutorial\/");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>